courses/free-courses-en.md

```diff
@@ -337,6 +337,7 @@
 * [Golang \| Gin HTTP Framework](https://www.youtube.com/playlist?list=PL3eAkoh7fypr8zrkiygiY1e9osoqjoV9w) - Pragmatic Reviews
 * [Gophercises: Free Coding Exercises for Budding Gophers](https://gophercises.com) - Jon Calhoun (email address *required*)
 * [Learn Go Programming - Golang Tutorial for Beginners](https://www.youtube.com/watch?v=YS4e4q9oBaU) - FreeCodeCamp
+* [Learn Go in 12 minutes](https://www.youtube.com/watch?v=C8LgvuEBraI) - Smooth tutorial to learn Go
```

Comment:
Following [CONTRIBUTING video platforms](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#platforms-and-access-notes):
> - YouTube videos. We usually don't link to individual YouTube videos unless they are more than an hour long and are structured like a course or a tutorial.

**14min is too short**

---

It also is nice use right name as instructor. <del>`Smooth tutorial to learn Go`</del> :heavy_check_mark:  Jake Wright

Reply:
Sorry for that.
I already changed for one I watched in the past, it lasts 53 minutes.
I hope this works.

Reply:
Remember update pr fields according to

Reply:
I did a double check and I just edited to add more info.
Please let me know if I need to add something else.